src/Api/Search.php

```diff
@@ -390,11 +391,20 @@ public static function formatCatalogueOutput($catalogue)
                     $imageUrl = $course['imageUrl'];
                 }
 
+                $summary = "";
+                if ($course['teaserDescription'] != "" && $course['teaserDescription'] != null) {
```

Comment:
We need to check if `isset` before cheking if its empty as `teaserDescription` is only valid for Courses.

Reply:
no need it is always set as we are requesting it in graphql

Reply:
@alihabib-administrate its always set for courses not LPs

Reply:
yeah right